src/main/java/wolox/training/models/Book.java

```diff
@@ -132,6 +143,7 @@ public String getIsbn() {
     }
 
     public void setIsbn(String isbn) {
+        checkArgument(!Strings.isNullOrEmpty(isbn));
```

Comment:
After checking if it is null or empty, you can check if the content of the isbn field is a number.